psydac/api/ast/linalg.py

```diff
@@ -73,32 +74,32 @@ def toInteger(a):
 
 class LinearOperatorDot(SplBasic):
 
-    def __new__(cls, ndim, comm=None, **kwargs):
+    def __new__(cls, ndim, block_shape, comm=None, **kwargs):
```

Comment:
What is this `block_shape`?

Reply:
The block_shape gives the shape of the `BlockMatrix`

Reply:
What if this is not a  block matrix? Does `block_shape == (1, 1)` in this case?

Reply:
yes, when we have a StencilMatirx it will be equal to `(1,1)`

Reply:
I see. Please add this important detail to the docstring

Reply:
done in [3e97777](https://github.com/pyccel/psydac/pull/213/commits/3e97777c8291d486e1399ff285678d8b7695e4d4)

Reply:
Thanks! But the docstring should be added to the class `LinearOperatorDot`